android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/BridgeManagerProvider.java

```diff
@@ -113,6 +117,8 @@ private BridgeManagerProvider(@NonNull Context applicationContext) {
     @NonNull
     private final AccountDAO accountDAO;
     @NonNull
+    private final UploadDAO uploadDAO;
```

Comment:
Yea that makes sense, I'll clear all for now, but that is not the full story. When the user signs out, or a different user signs in, files waiting for upload and already have a UploadSession (pre-signed URL) should still get uploaded.